xpallets/assets/src/lib.rs

```diff
@@ -236,7 +237,7 @@ decl_module! {
             ensure_root(origin)?;
 
             let who = T::Lookup::lookup(who)?;
-            info!("[set_balance]|set balances by root|who:{:?}|id:{:}|balances_map:{:?}", who, id, balances);
+            info!("[set_balance] set balance by root, who:{:?}, id:{}, balances:{:?}", who, id, balances);
```

Comment:
Too much inconsistence nits like the following in this PR.

```suggestion
            info!("[set_balance] Set balance by root, who:{:?}, id:{}, balances:{:?}", who, id, balances);
```